### ISSUE TITLE
Qr encode location wrong when installing through brew

### ISSUE THREAD

**Issue by Graunephar** [martinschilliger/Alfred-QR-Code#1]
Thanks for sharing a great Alfred Workflow that I would never have though about myself. 

My qrencode ended up at "/opt/homebrew/bin/qrencode" after installation though brew. 

I used way to long time figuring this out.

Maybe a note in the readme would be nice. 

I used ´´´which qrencode´´´ to get the path for the script

**Comment by coreyms** [comment]
Yes, confirmed. I already knew of the new Brew install location and found quickly.

**Comment by D3Gamer** [comment]
This is still an issue, click on the script object and replace the path that is there with the path terminal gives you when you run "which qrencode" and the workflow will function.
